src/__tests__/QueryComplexity-test.ts

```diff
@@ -23,6 +23,23 @@ import {
 describe('QueryComplexity analysis', () => {
   const typeInfo = new TypeInfo(schema);
 
+  it('should calculate complexity', () => {
+    const ast = parse(`
+      query {
+        variableScalar(count: -100)
```

Comment:
Better use a value that is not an ignored edge case internally (negative values), so we can actually be sure the value is calculated.

Reply:
You're right, fixed.